docs/langchain.md

```diff
@@ -0,0 +1,94 @@
+# Deploying LangChain Agents on Fixie
+
+[LangChain](https://python.langchain.com/) is a popular framework for developing applications built on top of Large Language Models. Fixie supports LangChain as a first-class citizen. This means you can develop LangChain Agents/Chains/Prompts just as you typically would and then quickly deploy them onto Fixie infrastructure.
+
+## Why Deploy on Fixie?
+
+Deploying on Fixie gives you three major advantages:
+
+1. We host it for you (for free). Simply run `fixie deploy` and your Agent will be hosted in the cloud, complete with a REST API for calling it programatically and a shareable URL that will allow others to test your agent themselves.
```

Comment:
```suggestion
1. We host it for you (for free). Simply run `fixie deploy` and your Agent will be hosted in the cloud, complete with a REST API for calling it programatically and a shareable UI URL that will allow others to demo your agent in the browser.
```

We might want to make this a little more obviously separate from the programmatic access.

Reply:
"UI URL" is a bit wonky to me, but otherwise agreed!